### PR TITLE
Catch JHttp exections if we get the language list in backend

### PR DESCRIPTION
**Pull request by zero-24**
Pull Request for Issue #15034 

### Summary of Changes

Catch the JHttp exections if we get the language list in backend

### Testing Instructions

- go to this file: administrator\components\com_installer\models\languages.php
- add in line 115 `$updateSite = '';`
- go to Backend -> Manage -> Install Languages

### Expected result

Error message like:
image

### Actual result

image

### Documentation Changes Required

none

### Additional infos

Can somone look over the new string i added? Maybe there can be a better englisch used?

### CODE REVIEW

**Comment by zero-24** [comment]
better wording for the message:
image

File: administrator/language/en-GB/en-GB.com_installer.ini [hunk 1]
```diff
@@ -152,6 +152,7 @@ COM_INSTALLER_MSG_UPDATESITES_REBUILD_MESSAGE="Update sites have been rebuilt. N
 COM_INSTALLER_MSG_UPDATESITES_REBUILD_NOT_PERMITTED="Rebuilding update sites is not permitted."
 COM_INSTALLER_MSG_UPDATESITES_REBUILD_WARNING="Update sites have been rebuilt. No extension with updates sites discovered."
 COM_INSTALLER_MSG_UPDATESITES_REBUILD_SUCCESS="Update sites have been rebuilt from manifest files."
+COM_INSTALLER_MSG_WARNING_NO_LANGUAGES_UPDATESERVER="Your update server table seems to be not up-to-date. Please <a href="_QQ_"index.php?option=com_installer&view=updatesites"_QQ_" target="_QQ_"_blank"_QQ_">rebulid your update server table</a>"
 COM_INSTALLER_MSG_WARNINGFURTHERINFO="Further information on warnings"
```
Comment (infograf768) [inline administrator/language/en-GB/en-GB.com_installer.ini]: rebulid => rebuild. 😃

Comment (zero-24) [inline administrator/language/en-GB/en-GB.com_installer.ini]: fixed thanks

**Comment by brianteeman** [comment]
The update table is not up to date. Please ...

**Comment by chmst** [comment]
I have tested this item :white_check_mark: successfully on 86d99426b2236ec65c555d25f4b4756d221a79f7<br /><br />Thank you ;) @zero-24 <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15035">issues.joomla.org/tracker/joomla-cms/15035</a>.</sub>

**Comment by zero-24** [comment]
Thanks done @brianteeman

**Comment by franz-wohlkoenig** [comment]
I have tested this item :white_check_mark: successfully on 307e2834c4b19708f3e6cd534ae4c6e0654b1973<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15035">issues.joomla.org/tracker/joomla-cms/15035</a>.</sub>

**Comment by franz-wohlkoenig** [comment]
RTC after two successful testes.

**Comment by zero-24** [comment]
Thanks!

**Pull request merged**